tests/optimisation_test/algae_bot_recipe.yaml

```diff
@@ -0,0 +1,57 @@
+general:
+  id: 198
+  name: algae_bot_recipe
+process:
+- state_name: pickup
+  station:
+    id: 11
+    operation:
+      type: InputStationPickupOp
+    type: InputStation
+  transitions:
+    on_fail: end_state
+    on_success: dispense_liquid
+- state_name: dispense_liquid
+  station:
+    id: 12
+    operation:
+      properties:
+        dispense_info:
+          dye_A: 0.3
+          dye_B:
+          - 0.31
+          - 0.34
+          - 0.36
+          - 0.39
+          - 0.42
+          - 0.45
```

Comment:
This is fine but can it be replaced with list notation [....]? if it is too complicated or not possible, don't worry about it